hands-on.md

```diff
@@ -1,2 +1,68 @@
-
 # Description of hands-on demos/exercises
+
+## Approach
+
+The pedagogical approach of this workshop will be modeled on that of 
+the Software and Data Carpentry communities and will utilize the 
+following interactive components: 
+
+### Live Coding
+
+The content will be presented through a combination of speaking 
+and live coding in front of tutorial participants.  Presenters will 
+both talk about HPC concepts as described in the outline, and 
+demonstrate them in front of participants, with the expectation that 
+participants follow along and run the same commands as the presenter.  
+
+> **Example: Logging onto a large-scale computing system**
+> 
+> Presenter draws a diagram of a user's local computer and the 
+> login node of a remote computing system, and describes what is 
+> happening when users log in.  She then demonstrates 
+> logging into the provided cluster using the "ssh" command; all participants
+> use the same command to log in to the same cluster.  
+
+### Exercises
+
+In addition to the content presentation, the tutorial will contain 
+regular exercises where participants are expected to work individually 
+or in groups on a set of challenges.  There will be an exercise or 
+independent activity for about every 15-20 minutes of instruction.  
+
+> **Example: Submitting jobs to a batch scheduler**
+> 
+> Participants modify a job submission file 
```

Comment:
Sorry! I meant something like this maybe:

`submit-job.pbs` submits a request to run our script on the test file `test.csv`:

~~~
python analyze.py test.csv
~~~

Modify the script `submit-job.pbs` to instead submit a job to run the analysis on our actual input data, `temperatures.csv`. This job will need more than 1 GB, so you will also need to change the amount of memory that `submit-job.pbs` is requesting.